packages/hidp_django_admin/pyproject.toml

```diff
@@ -13,7 +13,7 @@ authors = [
     { name = "Robin van Leeuwen", email = "rvanleeuwen@leukeleu.nl" },
 ]
 dependencies = [
-    "django-hidp>=1.4.0,<2",
+    "django-hidp>=1.5.0,<2",
```

Comment:
Can stay on django-hidp>=1.4.0 unless there's some change that depends on something that changed in 1.5.0